### ISSUE TITLE
Error message during load-data-async

### ISSUE THREAD

**Issue by sebastian-nehrdich** [BuddhaNexus/buddhanexus#91]
Currently I get the following error-message when I load files with load-data-async: 

`Could not load file. Error:  [HTTP 409][ERR 1210] unique constraint violated - in index primary of type primary over '_key'; conflicting key: T03abhisamu`

However the loading of the file works OK. I think this error is only surfacing since we changed to graph-db, but I am not entirely sure.

**Comment by TrebuhD** [comment]
@sebastian-nehrdich are we certain this ID appears only once in the files?

**Comment by sebastian-nehrdich** [comment]
I checked the files and they are all unique. also they all just apear once in tib-files.json and once in tib-categories.json. 
I did a bit of debugging and found out that load_files_collection is causing the problem.  When I comment line 96 `load_files_collection(menu_file_json, segmentnrs, db)` out the error-message disappears. Seems this function wants to write something that is already in the DB. Maybe this rings a bell with you? Otherwise I will continue debugging tomorrow.
